addons/l10n_lt/data/account_chart_template_data.xml

```diff
@@ -0,0 +1,1495 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo noupdate="1">
+    <!-- Chart of Accounts Template -->
+    <record id="account_chart_template_lithuania" model="account.chart.template">
+        <field name="name">Lithuania - Accounting</field>
+        <field name="code_digits">1</field>
+        <field name="currency_id" ref="base.EUR"/>
+        <!-- Anglo-Saxon Accounting is used for reporting cost of good sold
+            when products are sold/delivered.-->
+        <field name="use_anglo_saxon" eval="True"/>
+        <!-- Languages for which the translations of templates could be
+        copied in the final object when generating them from templates -->
+        <field name="spoken_languages" eval="'lt_LT'"/>
+        <!-- Module account_default_journal_accounts handles default accounts for
+            Cash and Bank Journals. -->
+        <field name="bank_account_code_prefix">271</field>
+        <field name="cash_account_code_prefix">272</field>
+        <field name="transfer_account_code_prefix">273</field>
+    </record>
+    <!-- Account Templates -->
+    <record id="account_account_template_1130" model="account.account.template">
```

Comment:
Instead of defining `account.account.template` records in XML, it would be better to define them in CSV file for better performance

Reply:
FWIW, it is not possible to have `noupdate="1"` with CSV files (https://github.com/odoo/odoo/issues/35613).

Reply:
> If account.account.template records are not intended to be customized by the user after they are loaded, then I am all for CSV.

you got it right 👍

> As far as I know, there is no menu for user to even see those records. @rgo-odoo is that the case?

yes, there isn't any menu available for `account.account.template`